pkg/webhook/webhook.go

```diff
@@ -486,6 +513,16 @@ func MutateHandler(w http.ResponseWriter, req *http.Request) {
 					glog.Infof("network '%s/%s' doesn't use custom resources, skipping...", n.Namespace, n.Name)
 				}
 			}
+
+			/* parse the net-attach-def annotations for node selector labels and add it to the addedNsMap */
+			if ns, exists := networkAttachmentDefinition.ObjectMeta.Annotations[nodeSelectorKey]; exists {
+				nsNameValue := strings.Split(ns, "=")
+				if len(nsNameValue) == 2 {
```

Comment:
Shall we automatically append the label value if it's define with key only? for example:
In net-attach-def, the annotation is `k8s.v1.cni.cncf.io/nodeSelector: environment`, then NRI update it to `k8s.v1.cni.cncf.io/nodeSelector: environment=""`

Reply:
Good find! done.